CS157A-Team18/CS157A-Team18
# [D-49] Create Functions for Storing and Accessing Files on Firestore

Changes:
- Install `uuid` dependency for assigning unique file names in the future
- Create functions for storing and accessing files on Firestore
- Do prep work for adding a recipe to the DB

To Test:
1. Pull this branch
2. In the root directory, do `npm install`.
3. In the `client` directory, do `npm install`.
4. In the `client` directory, do `npm start`
5. When the browser opens up, navigate to http://localhost:3000/upload
6. At the bottom, upload a picture and click `Add Recipe`
7. Open the console. It should give back a url which you can access the picture:
![image](https://user-images.githubusercontent.com/36905285/69852715-d8ebf680-1239-11ea-9a76-2b3f869f8f73.png)

This may take a while depending on how long it takes to upload the picture
